### PR TITLE
Submission flow no consent keys Submission Screen (EXPOSUREAPP-4064)

### PR DESCRIPTION
**Pull request by AndroidMedaGalaxy**
### Description
Submission Flow No Consent Keys Submission Screen

### Steps to reproduce
The other UIs are being worked upon. once they're done you will be able to get to this screen after Ihr Testergibnis screen in No Consent Flow

### Screenshot
SubmissionKeysNoConsent

Note: 

1. Do not worry about the navigation from Submission keys to default thank you screen
The navigation will be changed while marrying the #1738 
2. CountryList is not being retrieved in the current environment. that's why you don't see any list being populated there.

Navigation Flow
Submission Keys -> close button on top -> home screen
Submission Keys - > press Einverstanden - > Google ENF API prompt will show up -> can observe progressbar below - > Thank you screen once completed

### TODO
Navigation to and from this screen. will be done in a seprate PR

### CODE REVIEW

**Review by janetback — DISMISSED** [review]
UA approved

**Review by ralfgehrer — COMMENTED** [review]
> CountryList is not being retrieved in the current environment. that's why you don't see any list being populated there.

Don't we have a default country in any case? E.g. Germany?

File: Corona-Warn-App/src/main/res/layout/include_submission_positive_other_warning_no_consent.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+    <data>
+
+        <variable
+            name="countryData"
+            type="java.util.List&lt;de.rki.coronawarnapp.ui.Country&gt;" />
+
+        <variable
+            name="viewModel"
+            type="de.rki.coronawarnapp.ui.submission.qrcode.consent.SubmissionConsentViewModel" />
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/layout/include_submission_positive_other_warning_no_consent.xml:15]: Please use the fragments's own view model

File: Corona-Warn-App/src/main/res/values-de/legal_strings.xml [hunk 2]
```diff
@@ -34,4 +34,6 @@
...
     <!-- YTXT: Body for your consent screen agreement share symptoms -->
     <string name="submission_your_consent_agreement_share_symptoms">"<b>Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese geteilt.</b>\n\nSie können Ihr Einverständnis zurücknehmen, indem Sie oben „Andere warnen“ deaktivieren."</string>
+    <!-- YTXT: Body for keys submission no consent text -->
+    <string name="submission_no_consent_your_consent_subsection_body">"Durch Antippen von „Einverstanden“ willigen Sie wie folgt ein: \n\n<b>Die App teilt Ihr Testergebnis, um Nutzer von offiziellen Corona-Apps, denen Sie begegnet sind, zu warnen. Die Warnung funktioniert in den oben genannten Ländern. Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese geteilt.</b> Bitte geben Sie Ihr Testergebnis (genauer gesagt: Ihrer Zufalls-IDs der letzten 14 Tage) im nächsten Schritt frei, indem Sie „Teilen“ antippen.\n\nSie können Ihr Einverständnis nach bereits erfolgter Warnung zurücknehmen, indem Sie die App löschen."</string>
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/values-de/legal_strings.xml:38]: There are too many line breaks, the space between paragraphs is too big.

**Comment by ritsam** [comment]
> > CountryList is not being retrieved in the current environment. that's why you don't see any list being populated there.
> 
> Don't we have a default country in any case? E.g. Germany?

If I keep the environment as INT the default country comes up. weird that it's not in WRU-XD. I'll investigate.

File: Corona-Warn-App/src/main/res/values-de/legal_strings.xml (hunk 2 above)
Comment (ritsam) [reply Corona-Warn-App/src/main/res/values-de/legal_strings.xml:30]: what do you recon here then? seperate texts for each line?

Comment (AlexanderAlferov) [reply Corona-Warn-App/src/main/res/values-de/legal_strings.xml]: I think line breaks are okay. We are using them in many screens for now.

File: Corona-Warn-App/src/main/res/navigation/nav_graph.xml [hunk 3]
```diff
@@ -247,6 +247,25 @@
...
+            app:argType="de.rki.coronawarnapp.submission.Symptoms" />
+        <action
+            android:id="@+id/action_submissionResultPositiveOtherWarningNoConsentFragment_to_mainFragment"
+            app:destination="@id/mainFragment"
+            app:popUpTo="@id/mainFragment"
+            app:popUpToInclusive="true" />
+        <action
+            android:id="@+id/action_submissionResultPositiveOtherWarningNoConsentFragment_to_submissionDoneFragment"
+            app:destination="@id/submissionDoneFragment"
+            app:popUpTo="@id/submissionDoneFragment"
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/navigation/nav_graph.xml:266]: You need to pop to homeFragment ( Inclusive = false) here to clear the back stack

**Review by AlexanderAlferov — APPROVED** [review]

File: Corona-Warn-App/src/main/res/navigation/nav_graph.xml (hunk 3 above)
Comment (ritsam) [reply Corona-Warn-App/src/main/res/navigation/nav_graph.xml]: Done

**Review by chiljamgossow — APPROVED** [review]

**Pull request merged**